renderer.js

```diff
@@ -1695,6 +2059,10 @@ class SDFGRenderer {
             if ('is_collapsed' in obj.attributes && !obj.type.endsWith('Exit'))
                 obj.attributes.is_collapsed = true;
         });
+
+        // Send updated SDFG to vscode extension
+        this.send_new_sdfg_to_vscode();
```

Comment:
Why do we do this when zooming to a view - or rather: when can the SDFG change based solely on a `zoom_to_view` action?